Teil 5

LJinzhen opened the pull request:
Diskussion

LJinzhen committed:
Create Teil5_diskussion

LJinzhen committed:
Update Teil5_diskussion

LJinzhen commented:
Wir haben den vierten Teil fertiggestellt, also führe bitte Teil4 in der  Branch aus, um unsere Ergebnisse zu sehen.

yiyi808 commented on Teil5_diskussion:
```diff
@@ -0,0 +1,313 @@
+library(vcd)
+
+#Fuer die ganze Daten
+Daten <- read.csv("~/GitHub/wissArbeiten_2122_Gruppe_11/Daten.csv")
```
Es tut uns leid, aber es gab ein Problem mit unseren ursprünglichen Daten und sie entsprachen nicht den Anforderungen für den Auftragstitel. Die überarbeitete Fassung wurde jetzt auf main hochgeladen.

yiyi808 reviewed (commented):
Es tut uns leid, aber es gab ein Problem mit unseren ursprünglichen Daten und sie entsprachen nicht den Anforderungen für den Auftragstitel. Die überarbeitete Fassung wurde jetzt auf main hochgeladen.

LJinzhen commented:
Ok.Dann ändern wir den vierten Teil wieder.

LJinzhen commented:
Wir werden die Änderung bald abschließen. Ist es okay, wenn wir das in einer halben Stunde besprechen?

yiyi808 commented:
Vielen Dank. Bis dann.

Haiyi11 commented:
OK, bis dann

Xinyi-Jing committed:
Update Teil5_diskussion

Xinyi-Jing committed:
Update Teil5_diskussion

Xinyi-Jing committed:
Update Teil5_diskussion

Xinyi-Jing committed:
Update Teil5_diskussion

LJinzhen committed:
Update Teil5_diskussion

Haiyi11 commented on Teil5_diskussion:
```diff
@@ -0,0 +1,313 @@
+#       haeufig.Mathe    haeufig.Statistik 
+#                0.20                 0.35 
+
+#35 Personen studieren Statistik.35 Personen studieren Data Science.
```
Die relativen Wahrscheinlichkeiten entsprechen dem, was wir festgelegt haben

LJinzhen committed:
Update Teil5_diskussion

yiyi808 committed:
Update Teil5_diskussion

Xinyi-Jing commented:
Jinzhen und ich haben fertig korrigierte .

Xinyi-Jing commented:
Jinzhen und ich haben fertig korrigiert .

Haiyi11 commented on Teil5_diskussion:
```diff
@@ -0,0 +1,315 @@
+#gibt es einen signifikanten Zusammenhang zwischen Studienfach
+#und Interesse_Math, d.h. die Studienfach ist nicht unabhaengig vom Interesse_Math.
+Mosaicplot_Math
+#####Mosaicplot Erklaerung zwischen Studienfach & Interesse_Math
```
Der Level des Interesses an Mathematik wird in drei Stufen unterteilt: hoch, mittel und niedrig; nur die hohen Stufen werden hier analysiert; die niedrigen Stufen und die Zusammenhänge zwischen den verschiedenen Fächern werden nicht berücksichtigt

Haiyi11 commented on Teil5_diskussion:
```diff
@@ -0,0 +1,315 @@
+#Insgesamt glaube ich , dass es keinen großen Unterschied in der Altersverteilung der vier Studienfächer gibt. 
+#Das erste Quartil, der Median und das dritte Quartil liegen alle sehr nahe beieinander.
+# Also anhand des Boxplots vermute ich, dass sich Alter und Studienfach nicht gegenseitig beeinflussen.
+
```
Hier fehlt eine Analyse, ob das Alter normalverteilt ist.

Haiyi11 replied:
Mit shapiro.test können wir prüfen, ob die Normalverteilung erfüllt ist. Aber die W-Statistik liegt nahe bei 1 und der p-Wert ist signifikant größer als 0,05, so dass nicht ausgeschlossen werden kann, dass es sich um eine Normalverteilung handelt. 
Wir verwenden dann die Zeichnung, um dies zu bestimmen. Durch qqnorm(Alter) und qqline(Alter) können wir sehen, dass die Daten einer Normalverteilung entsprechen.
Ich habe dem Code Folgendes hinzugefügt.

Haiyi11 committed:
Update Teil5_diskussion

Haiyi11 committed:
Rename Daten.R to Teil_1_Daten.R

Haiyi11 committed:
Rename Daten.csv to Teil_1_Daten.csv

Haiyi11 committed:
Rename Teil5_diskussion to Teil_5_Diskussion

yiyi808 committed:
Create Teil_4_Ideen.docx

Haiyi11 committed:
Delete Teil_4_Ideen.docx

Haiyi11 committed:
Rename Teil_1_Daten.csv to Teil_2_Daten.csv

Haiyi11 committed:
Rename Teil 4.R to Teil_4_Analyse.R

Haiyi11 committed:
Delete Re_Zusammenfassung.der.Ideen.docx

Haiyi11 committed:
Rename Teil_4_Analyse.R to Teil 4.R

Haiyi11 committed:
Rename Teil 4.R to Teil_4_Analyse.R

Haiyi11 committed:
Rename Teil_4_Analyse.R to Teil 4.R

yiyi808 committed:
Update Teil_5_Diskussion

Haiyi11 committed:
Create Teil_5_Zusammenfassung

 Nach dem Vergleich der beiden R-Code von Teil_1 und von Teil_4 haben wir das Problem zusammengefasst.

Haiyi11 committed:
Update Teil_5_Zusammenfassung

Xinyi-Jing committed:
Update Teil_5_Diskussion

Xinyi-Jing committed:
Update Teil 4.R

yiyi808 committed:
Update Teil_5_Zusammenfassung

Xinyi-Jing committed:
Korrigierte Teil 4

Haiyi11 committed:
Update Teil_5_Zusammenfassung

Haiyi11 commented on Teil_5_Diskussion:
```diff
@@ -0,0 +1,327 @@
+  ggtitle("Balkondiagramm des Interesses an der Mathe") +
+  theme_ipsum()
+
+e)Interesse_Prog
```
Analog zu Interest an Mathematik.

yiyi808 commented on Teil_5_Diskussion:
```diff
@@ -0,0 +1,327 @@
+#gibt es einen signifikanten Zusammenhang zwischen Studienfach
+#und Interesse_Math, d.h. die Studienfach ist nicht unabhaengig vom Interesse_Math.
+Mosaicplot_Math
+#####Mosaicplot Erklaerung zwischen Studienfach & Interesse_Math
```
Die Zusammenhängen auf Mathe_LK können nicht getrennt analysiert werden.

yiyi808 replied:
Ja. Die Zusammenhänge zwischen den drei Variablen lassen sich anhand des Mosaikdiagramms deutlich erkennen.
Das Mosaicdiagramm zeigt, dass es einen Teil des gelben Bereichs gibt, der eine große Fläche einnimmt, was bedeutet, dass Menschen, die Mathematik studieren und ein größeres Interesse an Mathematik haben, Mathematikunterricht nehmen.

Haiyi11 replied:
Ja. Die Zusammenhänge zwischen den drei Variablen lassen sich anhand des Mosaikdiagramms deutlich erkennen.
Das Mosaicdiagramm zeigt, dass es einen Teil des gelben Bereichs gibt, der eine große Fläche einnimmt, was bedeutet, dass Menschen, die Mathematik studieren und ein größeres Interesse an Mathematik haben, Mathematikunterricht nehmen.

Haiyi11 committed:
Update Teil_5_Zusammenfassung

yiyi808 commented on Teil_5_Diskussion:
```diff
@@ -0,0 +1,327 @@
+#gibt es einen signifikanten Zusammenhang zwischen Studienfach
+#und Interesse_Math, d.h. die Studienfach ist nicht unabhaengig vom Interesse_Math.
+Mosaicplot_Math
+#####Mosaicplot Erklaerung zwischen Studienfach & Interesse_Math
```
Beim Mosaikdiagramm stellen wir fest, dass die Daten hier einen Fehler aufweisen.

yiyi808 reviewed (commented):
Beim Mosaikdiagramm stellen wir fest, dass die Daten hier einen Fehler aufweisen.

yiyi808 committed:
Update Teil_5_Zusammenfassung

Haiyi11 committed:
Update Teil_5_Zusammenfassung

Haiyi11 committed:
Update Teil_5_Zusammenfassung

yiyi808 committed:
Update Teil_5_Zusammenfassung
